### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by StasOkhrym**
Screenshot 2022-10-24 153327
Screenshot 2022-10-24 153405
Screenshot 2022-10-24 154201
Screenshot 2022-10-24 153545
Screenshot 2022-10-24 153853

### CODE REVIEW

File: taxi/models.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=65, unique=True)
+
+    class Meta:
+        verbose_name = "Driver"
+        verbose_name_plural = "Drivers"
```
Comment (mltdll) [inline taxi/models.py:19]: It's probably better to make those namea lowercase.

**Review by mltdll — CHANGES_REQUESTED** [review]

**Review by mltdll — APPROVED** [review]

File: taxi/admin.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    search_fields = ["name", "country"]
+    list_display = ["name", "country"]
+    list_filter = ["country"]
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_display = ["model", "manufacturer"]
+    list_filter = ["model"]
+    search_fields = ["manufacturer"]
```
Comment (Nattalli) [inline taxi/admin.py:18]: you should make filter by `manufacturer` and search by `model`

**Review by Nattalli — CHANGES_REQUESTED** [review]

File: taxi/models.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    def __str__(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=65)
+    manufacturer = models.ForeignKey(
+        Manufacturer, related_name="cars", on_delete=models.CASCADE
+    )
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="drivers")
```
Comment (Nattalli) [inline taxi/models.py:30]: `related_name = cars` would be better here

**Review by Nattalli — CHANGES_REQUESTED** [review]

**Review by Nattalli — APPROVED** [review]
Good job!

**Pull request closed without merging**